modules/gdscript/gdscript_cache.cpp

```diff
@@ -251,6 +251,8 @@ Ref<GDScript> GDScriptCache::get_shallow_script(const String &p_path, Error &r_e
 		return singleton->shallow_gdscript_cache[p_path];
 	}
 
+	ERR_PRINT(vformat("GDScriptCache::get_shallow_script(%s)", p_path));
```

Comment:
Is this a debug print?

Reply:
Yes, there are still quite a few debug prints in several places so that I can see what gets called when. Order of these different calls was really important, and it still is as we're working on other regressions this "fix" apparently introduces 😅